packages/shared/src/components/profile/ActivitySection.tsx

```diff
@@ -42,7 +42,7 @@ export const ActivitySectionHeader = ({
         {title}
         <ActivitySectionSubTitle>
           {subtitle}
-          <ClickableText tag="a" className="ml-1" href={link}>
+          <ClickableText tag="a" target="_blank" className="ml-1" href={link}>
```

Comment:
I think there should be two buttons here right?

Reply:
This is a reused component so both of them do this now.